video/out/opengl/ra_gl.c

```diff
@@ -588,8 +588,21 @@ static bool gl_buf_poll(struct ra *ra, struct ra_buf *buf)
     return !buf_gl->fence;
 }
 
-static void gl_clear(struct ra *ra, struct ra_tex *dst, float color[4],
-                     struct mp_rect *scissor)
+static void gl_clear(struct ra *ra, struct ra_tex *dst, float color[4])
+{
+    GL *gl = ra_gl_get(ra);
+
+    assert(dst->params.blit_dst);
+    struct ra_tex_gl *dst_gl = dst->priv;
+
+    gl->BindFramebuffer(GL_FRAMEBUFFER, dst_gl->fbo);
+    gl->ClearColor(color[0], color[1], color[2], color[3]);
+    gl->Clear(GL_COLOR_BUFFER_BIT);
+    gl->BindFramebuffer(GL_FRAMEBUFFER, 0);
+}
```

Comment:
This code is just duplicated.

Maybe it would be better to remove `ra_fns.clear`, and instead have the backend choose a different implementation if the rc argument covers the entire framebuffer?